src/docs/product/user-feedback/index.mdx

```diff
@@ -1,25 +1,76 @@
 ---
 title: User Feedback
-sidebar_order: 120
+sidebar_order: 65
 description: "Learn how you can view user feedback submissions which, paired with the original event, give you additional insight into issues."
 ---
 
-While event data allows you to collect information about errors in your application, getting any extra helpful feedback can be crucial in determining the root cause of an issue. Sentry provides the ability to collect additional information with User Feedback.
+Sentry automatically detects errors thrown by your application, such as performance issues and user experience problems like rage clicks. But there are other frustrations your users may encounter (broken permission flows, broken links, typos, misleading UX, business logic flaws, and so on).
 
-The **User Feedback** page in [sentry.io](https://sentry.io) displays feedback submissions from users who experienced an error while using your application. Each submission includes the name and email address entered by the user, as well as their description of what happened to produce the error:
+That’s where Sentry’s User Feedback Widget and Crash-Report User Feedback features come in. Setting up one or both, lets you collect feedback from your end-users when they run into bugs in your application. Sentry then links that feedback to available rich debugging context, including errors, replays, and tags such as URL, allowing you to get to the root of the problem faster.
```

Comment:
```suggestion
That’s where Sentry’s User Feedback Form and Crash-Report Modal features come in. Setting up one or both, lets you collect feedback from your end-users when they run into bugs in your application. Sentry then links that feedback to available rich debugging context, including errors, replays, and tags such as URL, allowing you to get to the root of the problem faster.
```

Reply:
@ryan953 Did we agree on using Feedback Form? [[Link to Slack thread](https://sentry.slack.com/archives/C05J714QPRR/p1700565079612939?thread_ts=1700500804.973179&cid=C05J714QPRR)] [[Google Doc Thread](https://docs.google.com/document/d/19zQoGVOlSLrgjf7j2SSXz-znvoLgU1dAA7VR-qGLcwM/edit?disco=AAAA_rCNenI)]

Reply:
Will push the Modal changes